### PR TITLE
RELATED: SD-311 SD-312 disable stacking config at high level when there is one measure

### PR DESCRIPTION
**Pull request by naibinh**
## PR Checklists
- [x] Squash commits
- [x] Unit test is green
- [x] Storybook is green
- [x] AD test cafe is green
- [x] KD test cafe is green
---
## Related PRs
AD: https://github.com/gooddata/gdc-analytical-designer/pull/2098
KD: https://github.com/gooddata/gdc-dashboards/pull/1740

### CODE REVIEW

**Comment by naibinh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3507/ : SUCCESS in 21s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2807/ : SUCCESS in 13m 19s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1494/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3508/ : SUCCESS in 20s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/904/ : SUCCESS in 23m 38s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3541/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2834/ : SUCCESS in 4m 49s

File: src/helpers/optionalStacking/common.ts [hunk 1]
```diff
@@ -16,3 +19,13 @@ export function getViewByTwoAttributes(
     }
     return [viewBy] as VisualizationObject.IVisualizationAttribute[];
 }
+
+export function verifyStackingConfig(props: IColumnChartProps | IBarChartProps): IChartConfig {
```
Comment (no23reason) [inline src/helpers/optionalStacking/common.ts]: The name of the function is a bit misleading. In my opinion, `verifyX` should return true/false meaning the X is valid or not. This function however _gets_ (extracts) something from `X` and _changes_ (sanitizes) it, so I would consider something like `extractSanitizedStackingConfig` or `getSanitizedStackingConfig`.

Comment (naibinh) [reply src/helpers/optionalStacking/common.ts]: it's done in fixup commit :)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3545/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2837/ : SUCCESS in 3m 22s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/775/ : SUCCESS in 47s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2839/ : SUCCESS in 10m 33s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1512/ : UNSTABLE in 27m 27s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/757/ : SUCCESS in 23m 30s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/779/ : FAILURE in 34s
- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED
- gooddata-react-components-storybook-zuul-docker : SKIPPED
- gooddata-react-components-examples-zuul-docker : SKIPPED

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/781/ : SUCCESS in 43s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2856/ : SUCCESS in 8m 32s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1520/ : UNSTABLE in 38m 05s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/767/ : SUCCESS in 20m 49s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/782/ : SUCCESS in 59s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2858/ : SUCCESS in 3m 53s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1523/ : SUCCESS in 15m 48s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/768/ : SUCCESS in 20m 50s

**Pull request merged**